app/views/install/compose.phtml

```diff
@@ -188,7 +188,6 @@ services:
       - traefik.http.routers.appwrite_realtime_wss.rule=PathPrefix(`/v1/realtime`)
       - traefik.http.routers.appwrite_realtime_wss.service=appwrite_realtime
       - traefik.http.routers.appwrite_realtime_wss.tls=true
-      - traefik.http.routers.appwrite_realtime_wss.tls.certresolver=dns
```

Comment:
what's the reason for removing this?

Reply:
This is not being used. We have not declared a resolver named `dns` in the traefik configuration above. 
And traefik complains about it too.